go.mod

```diff
@@ -1,16 +1,33 @@
-module github.com/crozzy/clair-load-test
+module github.com/vishnuchalla/clair-load-test
```

Comment:
- Yeah, we shouldn't make elasticsearch a hard dependency. It should just get used if the information is provided.
- The makefile doesn't add anything -- all those things are single commands and they're all `PHONY` targets.
- Shuffling the code around doesn't actually help for those things, since they're used in exactly one place.

Reply:
You should fix the name before we merge this, then we don't have to change it again.